dependencies/dependencies.gradle.kts

```diff
@@ -25,5 +25,6 @@ dependencies {
 		api("com.tngtech.archunit:archunit-junit5-api:${Versions.archunit}")
 		api("com.tngtech.archunit:archunit-junit5-engine:${Versions.archunit}")
 		api("org.slf4j:slf4j-jdk14:${Versions.slf4j}")
+		api("org.jetbrains.kotlinx:kotlinx-coroutines-core:${Versions.coroutines}")
```

Comment:
Should this be `compileOnly`?

Reply:
This file used as BOM: https://github.com/junit-team/junit5/blob/master/junit-jupiter-engine/junit-jupiter-engine.gradle.kts#L11

Actual dependency added here: https://github.com/junit-team/junit5/pull/2042/files#diff-923b2e78200b0be136b71b3fec482d83R22

See previous comment: https://github.com/junit-team/junit5/pull/2042#discussion_r385882647

Reply:
🔥 Thanks for checking!